src/contract/deploy.rs

```diff
@@ -51,12 +54,22 @@ impl<T: Transport> Builder<T> {
         let options = self.options;
         let eth = self.eth;
         let abi = self.abi;
+        let code = code.into();
+
+        let mut code_hex = String::from_utf8(code).unwrap();
```

Comment:
Change the signture to be `V: Into<String>` to avoid unsafe conversions (`.unwrap()`)

Reply:
I wanted to support the current execute that accepted a slice.
But I'll change this

Reply:
I see, but this `.unwrap()` might be incorrect, I don't think we should do that (it assumes that the `Vec<u8>` is valid UTF8 which might not be the case). I rather think we should change the API.
Either via `Into<String>` or `AsRef<str>` (now I think the latter might be better)

Reply:
Ok, will do.